config.example.yml

```
@@ -23,6 +23,14 @@ database:
   # Database password.
   password: CHANGEME
 
+  # List of low-level database options that can be set to influence some Icinga DB internal default behaviours.
+  options:
+    # Enforce Galera cluster nodes strict cluster-wide causality checks before executing
```

```suggestion
    # Enforce Galera cluster nodes to perform strict cluster-wide causality checks before executing
```

Or some other predicate.